### DISCUSSION TITLE
Ways to set the 'ChatGPT transformation prompt'

### DISCUSSION THREAD

**Discussion by notDavid** [chrislemke/ChatFred#57] [General]
Hi there,

i was wondering, do i have to go to configure the workflow each time i want to change the `ChatGPT transformation prompt`, or can it be easily set somehow with a keyword / shortcut / ... ?

Thank you!

**Comment by chrislemke** [comment]
Hey @notDavid!
Sorry for the late answer. Yes, at the moment this is the only way. I have not come up with a better idea. Maybe you have some ideas?

**Reply by yarub123** [reply]
@chrislemke just curious how do I go about disabling the answer always popping up in large type when just using the regular "cf" option?
I have unchecked the loading indicator large type in the workflow config, that worked. But the answer still shows up in large type.

**Reply by chrislemke** [reply]
Hey @yarub123!

You could use the "Paste" feature to paste the response to frontmost app. But this feature is only available for ChatGPT.

<img width="322" alt="Screenshot 2023-06-29 at 07 54 01" src="https://github.com/chrislemke/ChatFred/assets/11752694/66cd37a6-27c3-4d34-93fd-f988cab94348">

**Comment by April-June-August** [comment]
In fact there IS a way to set Configuration for the Workflow from within itself. Check it out [here](https://www.alfredforum.com/topic/19324-modifying-user-configuration-variable-from-within-the-workflow/). I'm considering working on it. 

Not sure when but you should be able to change Configurations like transformation prompt, jailbreak prompt, system prompt, LLM model choice using a keyword by then.
